utilities/GitUtilities.groovy

```diff
@@ -174,9 +176,10 @@ def getFileCurrentGitHash(String gitDir, String filePath) {
  * Returns the current Git url
  *
  * @param  String gitDir  		Local Git repository directory
- * @return String gitUrl       The current Git url
+ * @return String gitUrl        The current masked Git url
  */
 def getCurrentGitUrl(String gitDir) {
+	String retVal = new String()
```

Comment:
Just to be picky, but for readability, can you change the name of the variable to something else than `retVal`? Maybe `maskedGitUrl`?